packages/circuit-ui/components/Table/components/TableRow/TableRow.spec.tsx

```diff
@@ -50,7 +50,10 @@ describe('TableRow', () => {
     render(<TableRow onClick={onClick}>{children}</TableRow>);
     const rowEl = screen.getByRole('row');
 
-    rowEl.focus();
+    await userEvent.keyboard('{Tab}');
+
+    expect(rowEl).toHaveFocus();
```

Comment:
This better mimics how users would interact with the component. An element that's programmatically focusable might not be user-focusable.